apps/balancer-tools/src/components/Badge.tsx

```diff
@@ -1,27 +1,60 @@
 import cn from "clsx";
+import React from "react";
```

Comment:
good stuff 👍 for another reference, I also like how shadcn does this using CVA (which I think we should adopt for our base components): https://ui.shadcn.com/docs/components/badge

Reply:
will add this on my todo to not block this PR any longer c: